Rakefile

```diff
@@ -78,6 +38,25 @@ task :default => 'test'
 
 private
 
+def run_tests(scheme, sdk, destination)
+  sim_destination = "-destination \'#{destination}\'"
+  sh("xcodebuild -workspace PubNub.xcworkspace -scheme '#{scheme}' -sdk '#{sdk}' #{sim_destination} -configuration 'Debug' clean test | xcpretty -c ; exit ${PIPESTATUS[0]}") rescue nil
```

Comment:
Should we try run our tests with _Release_ configuration? There is set of optimisations performed on _Release_ configuration and to ensure what they won't expose any unexpected behaviour I suggest to run on _Release_ configuration. @jzucker2 What do you think?